docker-compose.override.yaml

```diff
@@ -3,10 +3,7 @@ version: '3'
 services:
     composer:
         volumes:
-            - ./config/:/app/config/
-            - ./composer.json:/app/composer.json
-            - ./composer.lock:/app/composer.lock
-            - ./symfony.lock:/app/symfony.lock
+            - ./:/app
```

Comment:
It will. Not currently used by anything, but for safety probably worth doing what we do with the `fpm` image: https://github.com/libero/browser/blob/e03ee59695fe7271b00ec0ee9fb0756d69401f62/docker-compose.override.yaml#L14

Volumes inside volumes work as expected, so in the `fpm` image the root is from the host, `vendor` is shared with the `composer` image and `var` is not shared with anything. _Except_ future runs, which break if you don't clear it (this is the problem mentioned in https://github.com/libero/browser/pull/4#discussion_r209846610).

Reply:
Makes sense 👍

Reply:
Is it worth adding in a `.dockerignore` file: https://docs.docker.com/engine/reference/builder/#dockerignore-file now that we're mounting the root?

Reply:
Unfortunately doesn't apply to volumes.

Reply:
ah its only for `COPY` in the Dockerfile

Reply:
IIRC for the build context, everything that is zipped up and sent to the Docker daemon during the build phase